dist/common/scripts/scylla_io_setup

```diff
@@ -142,6 +142,39 @@ if __name__ == "__main__":
             yaml.dump({ "disks": [ disk_properties ] }, properties_file,  default_flow_style=False)
             ioconf = open(etcdir() + "/scylla.d/io.conf", "w")
             ioconf.write("SEASTAR_IO=\"--io-properties-file={}\"\n".format(properties_file.name))
+        elif gcp_instance().is_gce_instance() and (is_debian_variant() or is_redhat_variant()):
+            idata = gcp_instance()
 
+            if idata.is_recommended_instance():
+                disk_properties = {}
+                disk_properties["mountpoint"] = datadir()
+                nr_disks = idata.nvmeDiskCount
+                if nr_disks >= 1 & nr_disks < 4:
+                    disk_properties["read_iops"] = 179861 * nr_disks
+                    disk_properties["read_bandwidth"] = 737304512 * nr_disks
+                    disk_properties["write_iops"] = 99963 * nr_disks
+                    disk_properties["write_bandwidth"] = 409628288 * nr_disks
```

Comment:
How constant are these values? Please run iotune on ~10 instances in a few regions to see that we don't get surprises.

Reply:
as per test docs https://docs.google.com/spreadsheets/d/1lMsGTfX0HQURH1L-A8czQXHhwLARQHBjXUDNWoEwrl4/edit#gid=2122362307
this was run on each instances, test repeated 5x, smallest number picked
However the variation between retests was around 7% for some of test repeats, which I think is partially due to Pavels fix for seastar (https://github.com/scylladb/seastar/commit/83919516c98deb78de4e6d909bb4468426c7836d (iotune: Report disk IOPS instead of kernel IOPS) ) was missing when the tests were done, it makes sense for me to do a quick retest of these numbers

Reply:
I only tested in us-east1-d region, I can try EU regions